match/lib/match.rb

```diff
@@ -21,30 +21,18 @@ module Match
   DESCRIPTION = "Easily sync your certificates and profiles across your team using git"
 
   def self.environments
-    envs = %w(appstore adhoc development)
-    envs << "enterprise" if self.enterprise?
-    return envs
-  end
-
-  # @return [Boolean] returns true if the unsupported enterprise mode should be enabled
-  def self.enterprise?
```

Comment:
Great point, however _match_ isn't designed as a tool to access methods and classes directly from the outside, as opposed to _spaceship_